inc/functions.php

```diff
@@ -6690,6 +6686,21 @@ function build_timezone_select($name, $selected=0, $short=false)
 		"13" => $lang->timezone_gmt_1300,
 		"14" => $lang->timezone_gmt_1400
 	);
+	return $timezones;
+}
+
+/**
+ * Build a time zone selection list.
+ *
+ * @param string The name of the select
+ * @param int The selected time zone (defaults to GMT)
+ * @param boolean True to generate a "short" list with just timezone and current time
```

Comment:
This function returned and returns a string too, so `@return` should be readded.

Except that it looks ready to merge.